### PR TITLE
Add Swift Package Manager support

### LINKED ISSUE #295: SPM support
Did you plan to support Swift Package Manager integrated into Xcode 11?

### PR DESCRIPTION
**Pull request by woxtu**
Hi,

This PR contains the following changes to add SPM support:

- Add the package manifest
- Add objc names to resources to load correctly in SPM (See [this forum](https://forums.swift.org/t/can-a-swift-package-include-a-table-view/40498) for details)

Close #295

### CODE REVIEW

**Comment by mrgrauel** [comment]
@sberrevoets do you like it? I'd like to see SPM support.

**Comment by mbuchetics** [comment]
bump 👍

File: Source/Actions/ActionCell.swift [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import UIKit
 
+@objc(SDCActionCell)
```
Comment (sberrevoets) [inline Source/Actions/ActionCell.swift:3]: Is this change related somehow?

**Comment by sberrevoets** [comment]
Sorry for the delay - this looks good. I'll look at bumping the version number this weekend.

Thanks for the contribution!

**Comment by mbuchetics** [comment]
awesome, could you release a new version?

**Comment by sberrevoets** [comment]
Should be up now!

**Comment by rowant1990** [comment]
Hey, I'm getting 'Invalid semantic version string '12.0' when trying to use this with SPM. Any chance you can adopt semantic versioning as suggested in the Apple docs https://developer.apple.com/documentation/xcode/publishing_a_swift_package_with_xcode

**Comment by sberrevoets** [comment]
My bad, should be fixed now.
